standalone-packages/vscode-textmate/src/onigLibs.ts

```diff
@@ -1,7 +1,8 @@
 /*---------------------------------------------------------
```

Comment:
Could you revert this file to master? This one needs to stay in sync with VSCode.

Reply:
Done, no idea what happened